app/models/post.rb

```diff
@@ -1,3 +1,17 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  after_save :post_increment
+
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
+
+  def recent_comment
+    comments.where(author_id.params[:id]).order(created_at: :desc).limit(5)
+  end
```

Comment:
### 🛑 Required change

- [ ] The comments returned should not be specific to an author/user. Kindly refer to the suggestion below

```ruby
def recent_comments
  comments.order('created_at Desc').limit(5)
end
```

Reply:
Thanks for your review, suggested changes implemented.